sections/discussion/author-topic.tex

```diff
@@ -0,0 +1,81 @@
+In this model, the author-topic distribution defines the probabilities of topics being written by a specific author.
+Then, just as \citet{author_topic_2012}, the similarity of authors can be found by calculating the symmetric Kullback-Leibler divergence:
+
+\begin{equation} \label{eq:author_similarity}
+	sKL(i,j) = \sum_{t=1}^{T}\left[\theta_{it}\, log \frac{\theta_{it}}{\theta_{jt}} + \theta_{jt}\, log \frac{\theta_{jt}}{\theta_{it}}\right]
+\end{equation}
+\noindent where $\theta_{it}$ is the probability of author $i$ having written about topic $t$, and the same for $\theta_{jt}$ with author $j$.
+
+In the context of using these similarities to assist Nordjyske, knowing how similar authors are, gives the opportunity to recommend new authors to readers, while the articles are about similar topics.
+In \autoref{tab:author_similarity}, the top 10 author pairs, based on this similarity measure, are shown.
+A smaller KL value means the authors are more similar.
+The number in parenthesis next to each author is the number of articles they have written in our dataset.
+
+\begin{table}[h]
+	\centering
+	\caption{Top 10 author pairs based on the symmetric KL divergence between authors.}
+	\begin{tabular}{r|c}
+		Author pair & KL \\
+		\midrule
+		Lars Termansen (328) \& Mikkel Færgemann Viken (91) & 1.50 \\
+		Morten Nis Klenø (17) \& Anne Helene Thomsen (606) & 1.72 \\
+		Lars Termansen (328) \& Lars Christensen (1293) & 2.43 \\
+		Esben Heine Pedersen (1689) \& Caspar Birk (71) & 2.47 \\
+		Lars Christensen (1293) \& Poul Christoffersen (65) & 2.53 \\
+		Lone Beck (92) \& Max Melgaard (587) & 2.74 \\
+		HANNE Lindblad Jensen (27) \& Peter Tordrup Larsen (5002) & 2.94 \\
+		Søren Kjær (95) \& Carl Chr. Madsen (785) & 2.98 \\
+		Heidi Majgaard B. Pedersen (244) \& Lisbeth Helleskov (361) & 3.05 \\
+		Lars Termansen (328) \& Morten Lind (413) & 3.16 \\
+		\midrule
+		Maximum & 34.51 \\
+		Median & 24.20 \\
+	\end{tabular}
+	\label{tab:author_similarity}
+\end{table}
+
+In general for these pairs, there does not seem to be a correlation between a high similarity and the categories of the articles they have written.
+While one author in a pair might have written for the sports category (Sport-avis) the other author might not have written for this category at all.
+This can also be seen for categories that cover geographic locations, where one author might have written for Aalborg (Aalborg-avis) and the other author can have written for Thisted (Thisted-avis).
+
+When looking at a sample of documents for the most similar author pair (Lars Termansen \& Mikkel Færgemann Viken), it is seen that they both write a mix of regular news and sports articles.
+The reason why they become this similar, might be that the ratio between news and sports news for both authors is similar, and possibly also because of the types of news they write about.
```

Comment:
```suggestion
The reason why they become this similar might be that the ratio between news and sports news for both authors is similar, and possibly also because of the types of news they write about.
```

Reply:
```suggestion
The reason for their high similarity, might be that the ratio between news and sports news for both authors is similar, and possibly also because of the types of news they write about.
```

Reply:
```suggestion
Their high similarity could be due to the ratio between news and sports news for both authors is similar, and possibly also because of the types of news they write about.
```